batcher/aligned-batcher/src/connection.rs

```diff
@@ -14,41 +14,50 @@ use tokio_tungstenite::{
     WebSocketStream,
 };
 
-use crate::types::{batch_queue::BatchQueueEntry, errors::BatcherError};
+use crate::{
+    types::{batch_queue::BatchQueueEntry, errors::BatcherError},
+    Batcher,
+};
 
 pub(crate) type WsMessageSink = Arc<RwLock<SplitSink<WebSocketStream<TcpStream>, Message>>>;
 
-pub(crate) async fn send_batch_inclusion_data_responses(
-    finalized_batch: Vec<BatchQueueEntry>,
-    batch_merkle_tree: &MerkleTree<VerificationCommitmentBatch>,
-) -> Result<(), BatcherError> {
-    for (vd_batch_idx, entry) in finalized_batch.iter().enumerate() {
-        let batch_inclusion_data = BatchInclusionData::new(vd_batch_idx, batch_merkle_tree);
-        let response = ResponseMessage::BatchInclusionData(batch_inclusion_data);
+impl Batcher {
+    pub(crate) async fn send_batch_inclusion_data_responses(
```

Comment:
I would move this function to lib.rs

Reply:
I agree, it is better to have all the batcher methods in one place

Reply:
Updated!